src/Build/BuildCop/Infrastructure/ConfigurationProvider.cs

```diff
@@ -19,56 +20,10 @@ namespace Microsoft.Build.BuildCop.Infrastructure;
 //  Let's flip form statics to instance, with exposed interface (so that we can easily swap implementations)
 internal static class ConfigurationProvider
```

Comment:
Let's turn this into an instance with exposed interface. This should make the testing of individual components easier.

(btw. it was my mistake to left this as static from the prototyping phase)

Reply:
Converted to the instance, interface removed for now, since we want to use the api internally for now.